docs/guides/tutorial.md

```diff
@@ -45,58 +45,57 @@ Then we need to create a fresh Rails application as following.
 First be sure to run `rails -v` and check you are using Rails 5.1.3 or above. If you are using an older version of Rails, you'll need to install webpacker with react per the instructions [here](https://github.com/rails/webpacker).
 
 ```bash
-cd <directory where you want to create your new Rails app>
+# For Rails 6.x
+rails new test-react-on-rails --skip-javascript
 
-# Any name you like for the rails app
-# Skip javascript so will add that next and get the current version
-# This is for Rails 7
+# For Rails 7.x
 rails new test-react-on-rails --skip-turbolinks --skip-javascript
 
 cd test-react-on-rails
 ```
 
 ## Add the shakapacker and react_on_rails gems
-To avoid issues regarding inconsistent gem and npm versions, you should specify the exact versions
+We recomment to use latest version of these gems. Otherwise, specify the exact versions
```

Comment:
@ahangarha please spellcheck in your editor! 

`recommend`

Reply:
My bad. Sorry for making the commit without proof check. Will update the word and even the sentence in a while.